src/api/app/models/workflow/step/configure_repositories.rb

```diff
@@ -12,18 +12,25 @@ def call(_options = {})
     target_project = Project.get_by_name(target_project_name)
     Pundit.authorize(@token.user, target_project, :update?)
 
+    repository_names = []
+
     step_instructions[:repositories].each do |repository_instructions|
       repository = Repository.includes(:architectures).find_or_create_by(name: repository_instructions[:name], project: target_project)
 
       target_repository = Repository.find_by_project_and_name(repository_instructions[:target_project], repository_instructions[:target_repository])
       repository.path_elements.find_or_create_by(link: target_repository)
 
       repository.repository_architectures.destroy_all
+      repository_names << repository_instructions[:name]
 
       repository_instructions[:architectures].uniq.each do |architecture_name|
         repository.architectures << @supported_architectures.select { |architecture| architecture.name == architecture_name }
       end
     end
+
+    # We have to store the changes on the backend
+    target_project.store(comment: "Added the following repositories to the project: #{repository_names.to_sentence}",
```

Comment:
Storing repository names in a variable isn't necessary. This can be done with:

```suggestion
    target_project.store(comment: "Added the following repositories to the project: #{step_instructions[:repositories].pluck(:name).compact.to_sentence}",
```

`compact` is to remove any potential `nil` values.

Reply:
@dmarcoux done